src/workspaces/com/fulcrologic/fulcro/cards/nested_dynamic_routing_tree_cards.cljs

```diff
@@ -108,16 +109,28 @@
 
 (def ui-a (comp/factory A {:keyfn :id}))
 
-(defrouter RootRouter [this props]
-  {:router-targets [A B]})
+(defrouter RootRouter [this {:keys [route-factory route-props]}]
```

Comment:
Might consider making a new card so the old test card is still available for quick checking the non-hooks